test/integration/run-tests-all-clouds.sh

```diff
@@ -24,4 +24,9 @@ if [[ "$#" -gt 0 ]]; then
   cloud="$1"
 fi
 
-test/integration/run-test.sh 'tcp://127.0.0.1:10000' "$nodeid" "$cloud"
+version='v1'
+if [[ "$#" -gt 1 ]]; then
```

Comment:
should add a new `pull-azuredisk-csi-driver-integration-v2` test pipeline in test-infra first

Reply:
I think @mukhoakash would be familiar with this. Could you help? Thanks.